### ISSUE TITLE
e: is tired; set: is wired.

### ISSUE THREAD

**Issue by NilsEnevoldsen** [scryfall/feedback#719]
This is a personal preference, but I think Scryfall should guide users to use the `set:` and `s:` keywords over the `edition:` and `e:` keywords.

`set:` is shorter, more accurate, and more intuitive than `edition:`. It's short enough that you might even push the non-abbreviated keyword for clarity.

Specifically:

* Change example searches in https://scryfall.com/docs/reference to use `set:` or `s:`
* Change set searches from https://scryfall.com/advanced to use `set:` or `s:`

**Comment by spacemonaut** [comment]
Is there a final difference? I've always used `e:` for sets.

**Comment by NilsEnevoldsen** [comment]
No. The keywords are synonyms. I think the `edition:` keyword began with magiccards.info. I have no idea why mc.info doesn't use `set:`.

**Comment by silasary** [comment]
It's a historical thing.

At once point, "sets" only referred to sealed product. Preconstructed decks
(Such as Duel Decks, or the Deckmasters/Beatdown Boxes) were not considered
"sets" (Despite having unique expansion symbols).
The thinking on this has now changed, and such products are called "Box
Sets", but MCI's terminology remains seated with the old terminology.

**Comment by spacemonaut** [comment]
So now we consider all of these things to be sets, but historically everything was an "edition" and only some things were sets?
